src/k8s/pkg/k8sd/types/cluster_config_certificates.go

```diff
@@ -21,6 +21,8 @@ func (c Certificates) GetAPIServerKubeletClientCert() string {
 func (c Certificates) GetAPIServerKubeletClientKey() string {
 	return getField(c.APIServerKubeletClientKey)
 }
+
+// Check if every field of the Certificate struct is Nil, returns false otherwise.
```

Comment:
go docs start with the function name
 
```suggestion
// Empty returns true if all Certificate fields are unset.
```